### ISSUE TITLE
base16 shell support

### ISSUE THREAD

**Issue by ldelossa** [NLKNguyen/papercolor-theme#113]
Hello,

Searching threw github issues it looks like PaperColor does support base16. However I'm not quite sure how to get it to work with base16-shell. https://github.com/chriskempson/base16-shell

Is this currently possible?

**Comment by NLKNguyen** [comment]
Yes and no, not yet. PaperColor does support terminal colors, so if you `set t_Co=16` or `set t_Co=8` then the terminal colors will be used, but the colors might not map nicely. The way base16 utilize terminal 16-color is different from how I utilize.

An effort that I didn't start yet is to provide a sample theme config for some popular terminal emulator. That will be equivalent to base16-shell idea, so that might work.

A while ago, I tried out on Windows Command Prompt, https://github.com/NLKNguyen/papercolor-theme/issues/71#issuecomment-234128705
https://github.com/NLKNguyen/papercolor-16/tree/master/windows-command-prompt
But it's been a while, and I'm not sure if it even works now. This terminal is hardly popular, so no one cares.

In short, PaperColor is designed to downgrade to 16-color terminal colors, but someone has to design the terminal theme to map the colors correctly.

**Comment by jonleopard** [comment]
I would also love to PaperColor implemented to work with base16-shell! I'll look into what @NLKNguyen said about mapping the colors correctly.

**Comment by ldelossa** [comment]
Keep us posted, papercolor is IMO one of the best color schemes. Would love to have it work with my base16 setup

**Comment by NLKNguyen** [comment]
@jonleopard Yes, please keep us posted. This is my reference to how the 16-color is intended https://github.com/NLKNguyen/papercolor-theme/blob/master/DESIGN.md

**Comment by jonleopard** [comment]
Just wanted to give everyone a small update.

I just submitted [pull request](https://github.com/chriskempson/base16-schemes-source/pull/39) to  base16's scheme source [repo](https://github.com/chriskempson/base16-schemes-source). Once it's approved, I can then start working with the base16 builders. If all goes well with building the base16 scheme, it should be available through base16-shell within the month!

Cheers :)

**Comment by ldelossa** [comment]
@jonleopard you are an angel sent from above.

**Comment by jonleopard** [comment]
Hello everyone,

My pull request was finally accepted, so you can now access the papercolor theme through any base16 builder tool. I'm currently using [base16-builder-typescript](https://github.com/golf1052/base16-builder-typescript).

After building the theme, you should manually copy the script to the appropriate directory so you can enable it. In my case, I'm using [base16-shell](https://github.com/chriskempson/base16-shell), [base16-vim](https://github.com/chriskempson/base16-vim), as well as [base16-lightline](https://github.com/mike-hearn/base16-vim-lightline) plugin. Until those repos update their themes list to include papercolor, you'll have to do it manually in order to access the theme.

With that said, the theme doesn't look exactly correct with the color values I assigned.

Here's a screenshot of iterm:
![screen shot 2018-09-06 at 10 31 53](https://user-images.githubusercontent.com/2111234/45145923-1a6e2d00-b1c2-11e8-81ab-1b86c06254da.png)




And vim (ignore the top and bottom bars since they are separate plugins that don't talk with base 16 yet):
![screen shot 2018-09-06 at 10 33 35](https://user-images.githubusercontent.com/2111234/45145972-2c4fd000-b1c2-11e8-9f9d-c2dd1f8cd9e0.png)



As you can see, the colors are quite a bit off haha.

You can take a look and see how I've assigned the colors here: https://github.com/jonleopard/base16-papercolor-scheme/blob/master/papercolor-light.yaml



I grabbed the values from here:
https://github.com/NLKNguyen/papercolor-theme/blob/master/colors/PaperColor.vim#L39


Any help or advice would be great!

**Comment by jonleopard** [comment]
Gonna take another shot at this. @NLKNguyen, any chance I could get some pointers here?
